src/main/java/net/sf/rails/ui/swing/GameSetupController.java

```diff
@@ -219,14 +214,10 @@ public void actionPerformed(ActionEvent arg0) {
             File saveDirectory = new File(Config.get("save.directory"));
 
             // define recent files
-            SortedSet<File> recentFiles = new TreeSet<File>(new Comparator<File> (){
-                public int compare (File a, File b) {
-                    return ComparisonChain.start()
-                            .compare(b.lastModified(), a.lastModified())
-                            .compare(a.getName(), b.getName())
-                            .result();
-                }
-            });
+            SortedSet<File> recentFiles = new TreeSet<>((a, b) -> ComparisonChain.start()
+                    .compare(b.lastModified(), a.lastModified())
+                    .compare(a.getName(), b.getName())
+                    .result());
```

Comment:
Do we want to use `ComparisonChain` or `Comparator.comparing` with `Comparator.thenComparing`?

Reply:
Doesn't matter to me-- didn't write the original code using `ComparisonChain` so don't know if there is a preference

Reply:
I didn't see that this call was there before this PR. In this case I think it is fine if we let it remain as is. On the other hand I would like to see if it also works with build in Java features (i.e `Comparator.comparing` and `Comparator.thenComparing`) :)